connaisseur/notary_api.py

```diff
@@ -100,12 +100,11 @@ def get_trust_data(host: str, image: Image, role: TUFRole, token: str = None):
 def get_delegation_trust_data(
     host: str, image: Image, role: TUFRole, token: str = None
 ):
-    if os.environ.get("LOG_LEVEL", "INFO") == "DEBUG":
-        return get_trust_data(host, image, role, token)
-
     try:
         return get_trust_data(host, image, role, token)
-    except Exception:
+    except Exception as ex:
```

Comment:
What does this have to do with the bug you're fixing in the other file? -- I've seen this fix in three different PRs now (one by @annekebr). I'm not sure what's going on, but as long as you know, it's fine :-)

Reply:
ya well i figured i put it in here, as it was supposed to be done in #81 and since this is a fixup for that, well...